### PR TITLE
README Updates

### PR DESCRIPTION
**Pull request by helenvholmes**
## **This PR does the following:**
- Adds a table of contents to the README
- Rewords introduction + makes reference to large design system ecosystem
- Moves some content to the wiki and links to the relevant wiki page

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,22 +20,27 @@ Follow these steps for setup:
...
 
 Changes in the local Design System will now be rebuilt and exported automatically.
 
-## Using the Design System
+## Using the Design System in Your Product
 
 To use the Design System in your project:
 
 1. Install the design system `npm install @nypl/design-system-react-components`
 2. Import the styles in your main css file `@import '~@nypl/design-system-react-components/dist/app.css';`
```
Comment (hokei) [inline README.md]: `app.css` is the wrong filename.  It's `styles.css`.

File: README.md [hunk 2]
```diff
@@ -56,8 +61,6 @@ function NewComponent(props) {
 }
 ```
 
-### Storybook
-
-There should be no need to run the static Storybook instances. If the static Storybook npm script is run, make sure to not commit the `storybook/storybook-static` directory.
```
Comment (hokei) [inline README.md:61]: I think this note about deleting the `storybook-static` directory if you ever run static, should stay.

**Review by hokei — CHANGES_REQUESTED** [review]
yay better documentation!  Just some nits

File: README.md (hunk 2 above)
Comment (helenvholmes) [reply README.md:61]: I had moved it to the wiki, but I can move it back—it felt more like contributing-gotchas than top-level info

**Comment by helenvholmes** [comment]
@hokei There isn't any good way to request review on it, but I've tried to also clean up the wiki—I'd love your eyes on it and maybe we request Bill run through it to make sure it's shipshape

EDIT: Made #232 to capture this

**Review by hokei — APPROVED** [review]

**Pull request merged**